src/host/PtySignalInputThread.cpp

```diff
@@ -191,9 +204,7 @@ void PtySignalInputThread::CreatePseudoWindow()
             break;
         }
         default:
-        {
-            THROW_HR(E_UNEXPECTED);
-        }
+            break;

```

Comment:
We can't throw an exception here - no one is catching it and it would cross an ABI boundary otherwise.